frontend/src/pages/Match/index.js

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
```

Comment:
Hm how about such as grouping:
- `react` and `react-router-dom`
- Material UI and Material UI Icons
- internal components and CSS stylesheet

So that they do not take up too much space since they are just imports anyway?

Reply:
Yea, that looks good!